xs2a-adapter-service-api-adapter/src/main/java/de/adorsys/xs2a/adapter/adapter/CertificateSubjectClientIdOauth2Service.java

```diff
@@ -0,0 +1,45 @@
+package de.adorsys.xs2a.adapter.adapter;
+
+import de.adorsys.xs2a.adapter.http.UriBuilder;
+import de.adorsys.xs2a.adapter.service.Oauth2Service;
+import de.adorsys.xs2a.adapter.service.Pkcs12KeyStore;
+import de.adorsys.xs2a.adapter.service.model.TokenResponse;
+
+import java.io.IOException;
+import java.net.URI;
+import java.security.KeyStoreException;
+import java.util.Map;
+
+/**
+ * Sets client_id parameter to organization identifier from eIDAS certificate
```

Comment:
JavadocStyle: First sentence should end with a period.

Reply:
Fixed.